specs/merge/client-settings.md

```diff
@@ -19,3 +20,9 @@ To coordinate manual overrides to [`TERMINAL_TOTAL_DIFFICULTY`](./beacon-chain.m
 
 Except under exceptional scenarios, this setting is expected to not be used. Sufficient warning to the user about this exceptional configurable setting should be provided.
 
+### Override terminal block hash
+
+To allow for fork coordination around a specific PoW block, clients must also provide `--terminal-block-hash-override` as a configurable setting.
```

Comment:
suggest removing "fork" here. There is a hardfork in beacon chain logic and then a subsequent transition from pow to pos (something like a fork of course...) but suggest "transition" to avoid name collision in this process

```suggestion
To allow for transition coordination around a specific PoW block, clients must also provide `--terminal-block-hash-override` as a configurable setting.
```

Reply:
good point